dashboard/app/helpers/district_dropdown_constants.rb

```diff
@@ -0,0 +1,8 @@
+module DistrictDropdownConstants
```

Comment:
@aoby What happens when we start storing these values in User.rb as well?  And possibly start validating them for users too?  I chose this so that it was model-agnostic, but maybe there's a better place?